### DISCUSSION TITLE
Artnet

### DISCUSSION THREAD

**Discussion by protoxz** [kryops/vlight#1] [Q&A]
Hi, i wanted to use vlight, but i struggle with artnet config, my artnet device ip is 192.168.200.187, and the PC connected to the device is 192.168.200.1, how can i configure it ? thanks for your help

**Comment by kryops** [comment]
Hi @protoxz,

what does your `vlight-config.js` look like?
It should contain something like

```ts
enableArtNetDevices: true,
artnetHost: '192.168.200.187'
```

Unfortunately, I don't have any experience using artnet myself, but using https://github.com/cpvalente/stupidArtnet I could determine that the vlight universe is mapped to artnet's universe 0.

Otherwise you could try and play around with the options of the `artnet` module that vlight uses:

- Available options: https://www.npmjs.com/package/artnet#options
- Applied here: https://github.com/kryops/vlight/blob/main/backend/src/devices/artnet/index.ts#L29

**Comment by protoxz** [comment]
Hello,

I managed to make it work ! I edited the file https://github.com/kryops/vlight/blob/main/backend/src/devices/artnet/index.ts#L16 Line 16 

Before :  server.set(channel, value)

After : server.set(1, channel, value)

This is to set the universe, by default it's 0, maybe could you add it in the config file ?

Thanks for your help

**Reply by kryops** [reply]
Awesome, thanks for investigating! 🚀 

I added a new `artnetUniverse` config.
